.github/workflows/coverage.yml

```
@@ -16,14 +16,56 @@ jobs:
         name: Install Python
         with:
           python-version: '3.8'
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          profile: default
+          components: llvm-tools-preview
       - name: Install dependencies
-        run: pip install tox coveragepy-lcov
-      - name: Run coverage report
-        run: tox -ecoverage
+        run: |
+            pip install coveragepy-lcov
+            sudo apt-get install lcov
+      - name: Download grcov for rust coverage
+        run: curl -L https://github.com/mozilla/grcov/releases/download/v0.8.11/grcov-x86_64-unknown-linux-gnu.tar.bz2 | tar jxf -
```

`tar f -` seems unnecessary - `tar xj` presumably wouldn't need stdin specifying?

(but that's super minor)

---

I think I just copied this from an old version of the grcov docs (and took it from what I was doing in retworkx). But yeah looking at it now `tar xj` is definitely sufficient.

```suggestion
        run: curl -L https://github.com/mozilla/grcov/releases/download/v0.8.11/grcov-x86_64-unknown-linux-gnu.tar.bz2 | tar xj
```